tachyon/zk/lookup/halo2/prover_impl.h

```diff
@@ -230,10 +234,10 @@ void Prover<Poly, Evals>::Open(
         permuted_pairs_[i];
 
     openings.emplace_back(OPENING(grand_product_poly, x));
-    openings.emplace_back(OPENING(grand_product_poly, x_next));
     openings.emplace_back(OPENING(permuted_pair.input(), x));
-    openings.emplace_back(OPENING(permuted_pair.input(), x_prev));
     openings.emplace_back(OPENING(permuted_pair.table(), x));
+    openings.emplace_back(OPENING(permuted_pair.input(), x_prev));
+    openings.emplace_back(OPENING(grand_product_poly, x_next));
```

Comment:
Could you change this to original? This will be included in https://github.com/kroma-network/tachyon/pull/305